aten/src/ATen/native/quantized/cpu/qsort.cpp

```diff
@@ -0,0 +1,51 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/cpu/vec256/vec256.h>
+#include <ATen/native/SortingUtils.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
+#include <ATen/native/quantized/cpu/quantized_ops.h>
+
+#include <algorithm>
+
+namespace at {
+namespace native {
+
+std::tuple<Tensor&, Tensor&> quantized_topk_out_cpu(
```

Comment:
what are the semantics of _out version? do we generally use it's quantizer? We need to either enforce quantizer to be the same or reset quantize in the output. cc @jerryzh168  on what we usually support in other ops

Reply:
Hmm, good question. Exposing the _out variant implies doing a requantization in the kernel. I think I can just not expose the _out variant as an op and only use it internally, thus ensuring the quantizer for the input and output are the same

Reply:
`_out` variant is a way of passing output `scale` and `zero_point` right now.
the initial motivation is to reuse the API we already have in torch namespace, since many torch functions have an out variant. 
e.g.
```
add(a, b, out_scale, out_zp)
```
```
out = empty_quantized(out_scale, out_zp)
add_out(a, b, out)
```